pkg/scheduler/framework/plugins/volumebinding/binder.go

```diff
@@ -610,6 +628,89 @@ func (b *volumeBinder) bindAPIUpdate(ctx context.Context, pod *v1.Pod, bindings
 	return nil
 }
 
+// retryableUpdatePV will retry updating pv when any conflicts occur.
+func (b *volumeBinder) retryableUpdatePV(ctx context.Context, binding *BindingInfo) (latestAPIPV *v1.PersistentVolume, err error) {
+	logger := klog.FromContext(ctx)
+
+	// Updating binding.pv directly to maintain the same approach as before.
+	latestAPIPV, err = b.kubeClient.CoreV1().PersistentVolumes().Update(ctx, binding.pv, metav1.UpdateOptions{})
```

Comment:
Why not this part of the retry logic?

Reply:
thanks for reviews. @kerthcet 

oh, When updating PVC/PV for the first time, use the objs from cycleState directly. (keep currently impl, it will success for most cases) 

Only when there is an update conflict appears,  we start the retry logic.

But I'm not entirely sure if this makes sense, or is it more reasonable for us to directly enter the logic of retry?

Reply:
You can add a flag here, like `refresh`, if not true, get the latest obj and try again.

Reply:
I think directly enter retry logic is easier to understand, so I removed the logic about updating PVC/PV  from cycleState for the first time.

PTAL, thanks. @kerthcet  

please feel free to make any suggestions.